plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/style.scss

```diff
@@ -7,35 +7,19 @@
 	flex-wrap: wrap;
 }
 
-.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill {
-	border-width: 1px;
-	border-style: solid;
-	border-radius: 2px;
-	padding: 0.25em 0.75em;
-	font-size: var(--wp--preset--font-size--small);
+.wp-element-button.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill {
 	cursor: pointer;
 	user-select: none;
 
-	&:hover {
```

Comment:
We let the hover be handled by the wp-button-element. TODO: the hover on the outlined buttons.

Reply:
I think the outline button hover is something we can go without on a v1 of this block, we should find a way to deal with the element variation and hover styling in core